### ISSUE TITLE
Movie Requests by other user (non-admin) are not visible as Admin login

### ISSUE THREAD

**Issue by YodaSpow** [tidusjar/Ombi#2283]
#### Ombi build Version:

V 3.0.3330

#### Update Branch:

Master

#### Media Sever:

Plex

#### Media Server Version:

1.13.2.5102 

#### Operating System:

Apple macOS High Sierra 10.13.4
OS Architecture: X64
OS Description: Darwin 17.5.0 Darwin Kernel Version 17.5.0: Fri Apr 13 19:32:32 PDT 2018; root:xnu-4570.51.2~1/RELEASE_X86_64

#### Ombi Applicable Logs (from `/logs/` directory or the Admin page):

```

Not sure how to get these on macOS High Sierra 10.13.4

```

#### Problem Description:

1. Movie Requests by other user (non-admin) are not visible when logged in as Admin
2. If I login as one of my non-admin accounts I don't see the movie requests made there either
3. Have Settings>Ombi Configuration - "Hide requests from other users" unchecked **This what I want**
4. Only movie request I can see are the ones made on the Admin account on all user logins

#### Reproduction Steps:

Only way i found this out was I have set up new requests to be posted to Slack channel as a NON-Admin then realised the Movie was not visible in any logins (Admin or Non-admin account in question)
![1](https://user-images.githubusercontent.com/26332959/40633963-93721602-62ea-11e8-8ff9-2c5587359fda.png)

Its only visible if I then search without knowing about it, it shows who requested it but this should be visible in the main view of Movie Requests for all users.
![screen shot 2018-05-29 at 02 46 29](https://user-images.githubusercontent.com/26332959/40634077-37de2730-62eb-11e8-860f-a82d85c56f50.png)

**Comment by tidusjar** [comment]
So you was able to see it by using the searchbar?

**Comment by YodaSpow** [comment]
Hi @tidusjar I have cleared out some old requests now as I had a lot in there.
I have discovered that the request for `Shoplifters` is actually in there. 

My username is `spowart` and the user in question is `graham` which should logically be 1st in the list or last in the list

However is buried somewhere in the mid-pages, the the problem appears to be a sorting issue as I have a lot of active/pending requests it moves around so finding it is tricky.
